SPECS/shadow-utils/shadow-utils.spec

```diff
@@ -36,27 +36,27 @@ in a secure way.
 sed -i 's/groups$(EXEEXT) //' src/Makefile.in
 find man -name Makefile.in -exec sed -i 's/groups\.1 / /' {} \;
 sed -i -e 's@#ENCRYPT_METHOD DES@ENCRYPT_METHOD SHA512@' \
-    -e 's@/var/spool/mail@/var/mail@' etc/login.defs
+    -e 's@%{_var}/spool/mail@%{_var}/mail@' etc/login.defs
```

Comment:
I believe these macros should not be used for package directory paths in the build chroot (linter needs to change). They should only be used for paths on the host system (such as in the %files section).

Reply:
Yes- changes to the linter are coming to make it less aggressive in this regard.

Reply:
+1. And to be clear: the changes in the `%build`, `%install` and `%file` sections look good. In here we can leave the second macro (right after the second `@` symbol).

Reply:
Updated PR. 
Thomas can you help removing var macro check for linter.